src/ui/EditorTabManager.cpp

```diff
@@ -55,13 +54,34 @@ EditorTabManager::EditorBy(entry_ref* ref)
 		}
 
 		BEntry matchEntry(editor->FileRef(), true);
-
 		if (matchEntry == entry)
 			return editor;
 	}
 	return nullptr;
 }
 
+
+Editor*
+EditorTabManager::EditorBy(const node_ref* nodeRef) const
+{
+	int32 filesCount = CountTabs();
+	for (int32 index = 0; index < filesCount; index++) {
+		Editor* editor = EditorAt(index);
+		if (editor == nullptr) {
+			BString notification;
+			notification
+				<< "Index " << index << ": NULL editor pointer";
+			LogInfo(notification.String());
+			continue;
+		}
+
+		if (editor->NodeRef() == nodeRef)
```

Comment:
Shouldn't we compare by value here?

Reply:
That's a doubt I had, in fact

Reply:
To me yes, we should. We could also change the signature function to use a "const node_ref&" to avoid confusion.

Reply:
Wouldn't be a problem if we start supporting "empty" files ?

Reply:
Oh sorry, I got it wrong: you meant a node_ref& only for the EditorBy(), right ?

Reply:
Now I compare the values.
I didn't change the method signature yet, though